.github/workflows/graalpy-embedding-demo.yml

```diff
@@ -0,0 +1,33 @@
+name: graalpy-embedding-demo
+on:
+  push:
+    paths:
+      - 'graalpy-embedding-demo/**'
+      - '.github/workflows/graalpy-embedding-demo.yml'
+  # pull_request: (requires EE, which is unavailable in PRs)
+  schedule:
+    - cron: "0 0 1 * *" # run every month
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  run:
+    name: Run 'graalpy-embedding-demo'
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Please use `actions/checkout@v4`. That will remove the warning about "Node.js 16 actions are deprecated" (thanks @fniephaus for advise).  I'm going to update in all workflow files.

Reply:
Done